client/src/pages/Lobby.js

```diff
@@ -166,9 +166,9 @@ const Lobby = (props) => {
             handleClose={handleClose}
           />
           {!alert && creatorId && getUserLobbyCountFull() && (
-            <ContinueButton onClick={handleStartInterview} color='primary'>
-              Start
-            </ContinueButton>
+            <div onClick={handleStartInterview}>
```

Comment:
Do you think we do this enough to add an onClick to the CustomButton? Either way is fine I think

Reply:
I think this should be on the CustomButton - I was thinking we should use the spread operator in CustomButton so that we can pass all the props that already exist for Material-UI, so we don't have to reinvent the wheel. Take a look at this CustomButton example:
https://github.com/creativetimofficial/material-dashboard-react/blob/master/src/components/CustomButtons/Button.js

This is probably overkill for you, but look at ...rest as an example of passing data to the Material-ui button so you can have all the props already in material-ui's buttons for free

Reply:
I didn't know I could pass functions like that as a prop but it seems to be working. I'll go ahead and change it so that we can add onClick to CustomButton

Reply:
<img width="716" alt="Screen Shot 2020-11-27 at 12 35 57 PM" src="https://user-images.githubusercontent.com/61174647/100473772-197c8b00-30ad-11eb-833d-2c4d26a26b21.png">

Is something like this okay?

Reply:
I changed all my buttons and passing onClick as a prop seems to still work

Reply:
Are there any problems if you click on a button that doesn't have an onClick passed to it? Using the {...rest} is more flexible, but I suppose we probably won't be adding anything else to these buttons in the future

Reply:
It seems to still work even if you do not pass onClick as a prop! Is it okay if I push this PR up now. I'll be needing CustomButtons for my interview table.

Reply:
yup, go for it